packages/gatsby-theme-project-portal/src/components/SiteSearch.stories.tsx

```diff
@@ -104,7 +104,7 @@ const invalidSearches = async ({ canvasElement }) => {
 
   await userEvent.clear(searchInput)
 
-  await userEvent.type(searchInput, "____", {
+  await userEvent.type(searchInput, "--", {
```

Comment:
Can you say a bit about what this line is changing?

Reply:
This line should cause lunr to throw an error (similar to what we saw with a caret), and we should see the error message thrown in the p tag.